app/_hub/kong-inc/upstream-timeout/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Upstream Timeout
+publisher: Kong Inc.
+categories:
+  - traffic-control
+type: plugin
+
+desc: Set timeouts on routes and override service-level timeouts
+description: |
+  Use the Upstream Timeout plugin to configure route-specific timeouts.
+  This plugin overrides any service-level timeout settings.
+
+kong_version_compatibility:
+  enterprise_edition:
+    compatible: true
+
+cloud: true
+enterprise: true
+plus: false
+
+params:
+  name: upstream-timeout
+  service_id: false
+  consumer_id: false
+  route_id: true
+  protocols:
+    - http
+    - https
+  dbless_compatible: yes
+  config:
+    - name: connect_timeout
+      required: false
+      default: null
+      value_in_examples: 4000
+      datatype: integer
+      description: |
+        The timeout, in milliseconds, for establishing a connection to the upstream server.
+        Overrides the [service object `connect_timeout` setting](/gateway/latest/admin-api/#service-object).
+    - name: send_timeout
+      required: false
+      default: null
+      value_in_examples: 5000
+      datatype: integer
+      description: |
+        The timeout, in milliseconds, between two
+        successive write operations when sending a request to the upstream server.
+        Overrides the [service object `send_timeout` setting](/gateway/latest/admin-api/#service-object).
```

Comment:
That service object setting doesn't exist at that link. I looked for it in the docs and I couldn't find it. 

I looked at [typedefs.lua](https://github.com/Kong/kong-ee/blob/bce43a48869d885d7ea154240f4cb29bc0bde989/kong/db/schema/typedefs.lua#L337). And that makes me think that in this case, we aren't overriding the `send_timeout` setting (because it doesn't exist yet), but rather just adding a new timeout value to the service object.  So maybe we don't need to link to anything at all here.

Reply:
Fixed, it should've been `write_timeout` on the service side.